integration_tests.py

```diff
@@ -348,22 +348,22 @@ def install_server():
     """Install DIRAC in the server container."""
     _check_containers_running()
 
-    typer.secho("Running server installation", fg=c.GREEN)
-    base_cmd = _build_docker_cmd("server", tty=False)
-    subprocess.run(
-        base_cmd + ["bash", "/home/dirac/LocalRepo/TestCode/DIRAC/tests/CI/install_server.sh"],
-        check=True,
-    )
-
     # This runs a continuous loop that exports the config in yaml
     # for the diracx container to use
     typer.secho("Starting configuration export loop for diracx", fg=c.GREEN)
-    base_cmd = _build_docker_cmd("server", tty=False, daemon=True)
+    base_cmd = _build_docker_cmd("server", tty=False, daemon=True, use_root=True)
     subprocess.run(
         base_cmd + ["bash", "/home/dirac/LocalRepo/ALTERNATIVE_MODULES/DIRAC/tests/CI/exportCSLoop.sh"],
         check=True,
     )
 
+    typer.secho("Running server installation", fg=c.GREEN)
```

Comment:
Does it have an influence to swap the order here ? I would have assumed it hasn't

Reply:
The server install script uploads proxies to the proxy DB

Reply:
Sorry I don't see how this is related. The `exportCSLoop` script does not rely on any proxy, it takes the `cfg` file directly

Reply:
You have this the opposite way around, server install script needs the DiracX CS to be kept in sync.

After installing the databases the install server script calls dirac-login. At this point we need the new CS to have been updated already else the token exchange fails.

Reply:
Ah right !  We did not have this problem before as we were minting the token directly :man_facepalming: 
I need to push some refactoring anyway, so I'll add it as a comment